packages/faas-functions/update-questionnaire-statuses-faas/src/update-questionnaire-statuses.js

```diff
@@ -0,0 +1,44 @@
+const sql = require('mssql');
+const config = require('./config');
+
+module.exports = async (context) => {
+  try {
+    const sqlConfig = {
+      user: config.sqlServer.username,
+      password: config.sqlServer.password,
+      database: config.sqlServer.database,
+      server: config.sqlServer.server,
+      pool: {
+        max: 10,
```

Comment:
Should these not be pat of Config?

Reply:
No because this is an instance of config settings for the server.